timesketch/lib/analyzers/sigma_tagger.py

```diff
@@ -35,11 +35,21 @@ def run_sigma_rule(self, query, rule_name, tag_list = None):
         """
         return_fields = []
         tagged_events_counter = 0
+        # TODO: tune the below line to be more effective
         events = self.event_stream(
             query_string=query, return_fields=return_fields)
         for event in events:
-            event.add_tags(['sigma_{0:s}'.format(rule_name)])
+            ts_sigma_rule = event.source.get('ts_sigma_rule',[])
```

Comment:
nit, space after the comma

Reply:
another nit... this is a list, so a better name would be `ts_sigma_rules` (plural)